#Single tag Data retrieved From M6e

time   timestamp
string ID
int32  rssi
int32  phase
int32  frequency

# NOTE : WE ARE NOT READING TAG data!!
